src/ERC7802/libraries/LibERC7802.sol

```diff
@@ -0,0 +1,161 @@
+        mapping(bytes4 => bool) _supportedInterfaces;
+    }
+
+    /// @notice Return storage pointer to ERC7802Storage
+    /// @dev Inline assembly to point storage to a fixed slot.
+    /// @return s Pointer to storage struct.
+
+    function getStorage() internal pure returns (ERC7802Storage storage s) {
+        bytes32 position = STORAGE_POSITION;
+
+        assembly {
+            s.slot := position
+        }
+    }
+
+    // the bridge account must be a trusted account
+    /// @notice Internal crosschain mint logic. MUST be called only after validating caller.
+    /// @dev Increases totalSupply and recipient balance and emits CrosschainMint.
+    /// @param _account The account to mint tokens to.
+    /// @param _value The amount to mint.
+    function crooschainMint(address _account, uint256 _value) internal {
+        ERC7802Storage storage s = getStorage();
+
+        if (s._trustedBridges[msg.sender] == false) revert ERC7802InvalidBridgeAccount(msg.sender);
+        if (_account == address(0)) revert ERC7802InvalidReciever(address(0));
+
+        unchecked {
+            s.totalSupply += _value;
+            s.balanceOf[_account] += _value;
+        }
+
+        emit CrosschainMint(_account, _value, msg.sender);
+    }
+
+    /// @notice Internal crosschain burn logic. MUST be called only after validating caller.
+    /// @dev Decreases totalSupply and the `from` balance and emits CrosschainBurn.
+    /// @param _from The account to burn tokens from.
+    /// @param _value The amount to burn.
+    function crosschainBurn(address _from, uint256 _value) internal {
+        ERC7802Storage storage s = getStorage();
+
+        if (s._trustedBridges[msg.sender] == false) revert ERC7802InvalidBridgeAccount(msg.sender);
+        if (_from == address(0)) revert ERC7802InvalidReciever(address(0));
+
+        uint256 accountBalance = s.balanceOf[_from];
+
+        if (accountBalance < _value) revert ERC7802InsufficientBalance(_from, accountBalance, _value);
+
+        unchecked {
+            s.totalSupply -= _value;
+            s.balanceOf[_from] -= _value;
+        }
+
+        emit CrosschainBurn(_from, _value, msg.sender);
+    }
+
+    //ERC165 and 7805 supports
+    /// @notice Query whether an interface id is supported (ERC-165 style).
+    /// @param interfaceId The interface id to query.
+
+    function supportInterface(bytes4 interfaceId) internal {
+        ERC7802Storage storage s = getStorage();
+        if (interfaceId != 0x33331994 || interfaceId == 0x01ffc9a7) revert ERC7802InvalidInterfaceId(interfaceId);
+
+        s._supportedInterfaces[interfaceId] = true;
+
+        emit Interface(interfaceId);
+    }
+
+    // @notice Add a trusted bridge address. Owner-only.
+    /// @param _bridge The bridge address to add.
+    function addTrustedBridges(address _bridge) internal {
+        ERC7802Storage storage s = getStorage();
+
+        if (msg.sender != s.owner) revert ERC7802InvalidOwner(msg.sender);
```

Comment:
Please remove the owner variable from use in this library.  Authorization should not happen in libraries.  See here: https://compose.diamonds/docs/design/design-for-composition#facets--libraries